### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by XKoop7321** [ihaveamac/custom-install#67]
Hi so I wanted to know if this requires any homebrewing whatsoever because I'm a bit clueless about homebrewing so I wanted to know. 

Thank you

**Comment by arminolat** [comment]
Hi! 

As far as I'm aware, this does require you to have a hacked 3DS.

You can find an amazing guide on how to get from normal 3DS to having a Homebrew enabled console here:
https://3ds.hacks.guide/

Make sure to go through the FAQs as well.

After that, simply follow the guides in the custom-install github. It's really simple, just follow it step by step and read carefully. 

In 2022, there's really no reason why not to have a hacked 3DS anyway.

Best,
Olat

**Comment by Masamune3210** [comment]
It does, it at least requires sigpatches since the tickets won't be legit unless you use a Legit CIA

**Comment by ihaveamac** [comment]
It requires custom firmware in almost any case because you still need a method to install the tickets (that is what custom-install-finalize is for). Only in the case of installing games you already purchased from the eShop, or otherwise already have an installed ticket for, would it not need this extra step.

**Comment by Masamune3210** [comment]
Good point, forgot the last step, dont answer tickets when still waking up lol
